app/models/reports/bulk_qts_report.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+module Reports
+  class BulkQtsReport < TemplateClassCsv
```

Comment:
So for example, this would become `BulkRecommendReport` or something like that.

Reply:
I might call it an `export` or something too - we're removing this from the 'reports' section entirely.